runelite-client/src/main/java/net/runelite/client/plugins/config/PluginListItem.java

```diff
@@ -143,6 +156,38 @@ private PluginListItem(ConfigPanel configPanel, @Nullable Plugin plugin, @Nullab
 		JLabel nameLabel = new JLabel(name);
 		nameLabel.setForeground(Color.WHITE);
 
+		final JMenuItem wikiPageMenuItem = new JMenuItem();
+		final JPopupMenu popupMenu = new JPopupMenu();
+		wikiPageMenuItem.setText("Wiki");
+		wikiPageMenuItem.addActionListener(e -> LinkBrowser.browse(wikiUrl()));
+		popupMenu.setBorder(new EmptyBorder(5, 5, 5, 5));
+		popupMenu.add(wikiPageMenuItem);
+
+		nameLabel.addMouseListener(new MouseAdapter()
+		{
+			@Override
+			public void mouseClicked(MouseEvent mouseEvent)
+			{
+				Component source = (Component) mouseEvent.getSource();
+				Point location = MouseInfo.getPointerInfo().getLocation();
+				SwingUtilities.convertPointFromScreen(location, source);
+				popupMenu.show(source, location.x, location.y);
+			}
+
+			@Override
+			public void mouseEntered(MouseEvent mouseEvent)
+			{
+				PluginListItem.this.lastForeground = nameLabel.getForeground();
```

Comment:
you can just store lastForeground inside this anon impl right. that seems better

Reply:
That does seem better. Didn't know that was an option. Done!